Backend/model/model.py

```diff
@@ -1,4 +1,55 @@
-from pydantic import BaseModel, EmailStr, validator
+from pydantic import BaseModel, EmailStr, validator, Field
+
+
+class ParameterValidator:
+    def validate_name(cls, value):
+        if value.startswith(' '):
+            raise ValueError("Name can not start with space")
+        if value.endswith(' '):
+            raise ValueError("Name can not end with space")
+        if value.count(' ') > 2:
+            raise ValueError("Name must not contain more than Two space")
+        if not value.isalpha():
+            raise ValueError("Name should be alpha")
+        return value
+
+    def validate_age(cls, value):
+        for i in value:
+            if i.isdigit() and int(i) > 0:
+                continue
+            else:
+                raise ValueError("only digit accepted and value should be greater than zero")
+        return value
+
+    def validate_phone_number(cls, value):
+        if not value.isdigit():
+                raise ValueError("Value should be 10 digit")
+        return value
+
+    def validate_password(cls, value):
+        if len(value) == 0:
+            raise ValueError("This field should not be empty")
+        return value
+
+    def validate_gender(cls, value):
+        if not value.isalpha():
+            raise ValueError("Value should be alphabet")
+        return value
+
+    def validate_pincode(cls, value):
+        if not value.isdigit():
+            raise ValueError("Value should be 6 digit")
+        return value
+
+    def validate_taluka(cls, value):
+        if not value.isalpha():
+            raise ValueError("Value should be alphabet")
+        return value
+
+    def validate_location(cls, value):
```

Comment:
this method can be renamed and combined as validate_is_empty along with this method validate_password. So only one is enough